rust/Hash Maps and Sets/geometric_sequence_triplets.rs

```diff
@@ -0,0 +1,32 @@
+use std::collections::HashMap;
+
+fn geometric_sequence_triplets(nums: Vec<i32>, r: i32) -> i32 {
+    // Use HashMap to track frequencies, with default value of 0
```

Comment:
Please make the comment closer to the original comment
<img width="1206" height="250" alt="image" src="https://github.com/user-attachments/assets/37bb046b-0ae3-43fa-a06a-230fac56506d" />